src/api/app/helpers/webui/staging/workflow_helper.rb

```diff
@@ -69,7 +69,9 @@ def create_request_links(request, users_hash, groups_hash)
     css = 'review' if request[:missing_reviews].present?
     css = 'obsolete' if request[:state].in?(BsRequest::OBSOLETE_STATES)
     css += ' delete' if request[:request_type] == 'delete'
-    link_content = [request[:package]]
+    # special-case SLFO maintenance patchinfos for which the numbers
+    # in the package name are considered an implementation detail
+    link_content = [request[:package].match?(/patchinfo\.\d+\.\d+/) ? 'patchinfo' : request[:package]]
```

Comment:
You have thought about the use cases of patchinfo extensions inside all the other maintenance workflows right?

Reply:
To my knowledge, the other maintenance workflows do not name patchinfo packages anything else than "patchinfo", so they wouldn't be affected.
And this makes the display of patchinfos (in the stagings webUI) consistent with the legacy workflow (granted, we weren't using stagings before).